variables.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash
+
+#Promenade Variables
+DOCKER_VERSION=1.12.6-0ubuntu1~16.04.1
+PROMENADE_LOAD_IMAGE=$SCRIPT_DIR/promenade.tar
```

Comment:
I think we should avoid setting this variable.

It's just here to help speed iteration, so that I don't have to push each iteration image to quay, then download it 3 or 4 times.